src/frontend/utils/tidslinje.ts

```diff
@@ -1,15 +1,8 @@
+import { addDays, endOfMonth, isAfter, isBefore, startOfMonth, subDays } from 'date-fns';
+
 import type { Periode } from '@navikt/familie-tidslinje';
 
-import {
-    erEtter,
-    hentFørsteDagIYearMonth,
-    erFør,
-    hentSisteDagIYearMonth,
-    kalenderDatoTilDate,
-    leggTil,
-    KalenderEnhet,
-    trekkFra,
-} from './kalender';
+import { parseIsoString } from './dato';
 import type { IYtelsePeriode } from '../typer/beregning';
 
 export const splittYtelseVedEndringerPåAnnenYtelse = (
```

Comment:
Tror eeegentlig det hadde vært mer ryddig å gjort denne jobben backend.

Reply:
Hvorfor har vi den i det hele tatt? Ser ut som den brukes for å skille småbarnstillegg-andelene fra utvidet-andelene. De er da hver for seg backend? Har vi samlet dem før vi sender dem frontend for å splitte dem her? 

Det er kanskje litt ute av scope å se på i denne oppgaven, men synes hele denne funskjonen er rar.

Reply:
Det er ikke for å skille de fra hverandre på den måten, det er for at det skal opprettes en splitt i tidslinjen. Men enig at det hadde vært mer ryddig å gjøre dette backend, men da må man legge til et eget kall kun for det 🤷‍♂️ tror det blir en stor endring å fikse på noe her, så dropper å fikse det i denne PRen